package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "the-travel-shelf",
+  "version": "1.0.0",
+  "description": "A magazine and index of beautiful global independent bookstores",
+  "main": "index.js",
+
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/RichardLitt/the-travel-shelf.git"
+  },
+  "keywords": ["independent", "bookstores", "index"],
+
+  "author": "Richard Littauer <richard.littauer@gmail.com> (https://burntfen.com)",
+  "license": "MIT",
+  "contributors": ["Joanna Katsifaraki", "Jose Esplana <jmesplana@gmail.com> (www.jey-em.com)", "Hanie Jafari me@hjafari.com (www.hjafari.com)", "Vernon Lillies <vernon.lillies@gmail.com>" ],
```

Comment:
Hanie needs `<>` around her email. 

Can you look for a `package.json` linter for me?